receiver/splunksearchapireceiver/config.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	Username                string        `mapstructure:"splunk_username"`
 	Password                string        `mapstructure:"splunk_password"`
 	Searches                []Search      `mapstructure:"searches"`
+	EventBatchSize          int           `mapstructure:"event_batch_size"`
```

Comment:
Should this be on the search configuration itself?

Reply:
I guess. If this is something that a user would want to change for different queries, then sure.